hyahatiph-labs/gitpayd
# Reemuru/bug fixes

Closes #47 

### Description

Run prod tests on the old version without grpc
need to fix that comment and re-test in prod

### Payment Request

LN: lnbc1u1ps8y5qzpp5v5aqp5s8khyk6v2vxmakmah9vfakmughwym2azx8677hwhgyfvmsdqdg9jxgg8sn7f27cqzpgxqr23ssp5w8uxc3lueqzjrqdtk2xf69uhjg4zktf5ks6nzcdp9r630v75wzvq9qyyssql8qmvdz4qgewcff9k29f4pu3mdcsk4gan8d542p9tlnyezy0je59xus30yu7tzhcc2hlehwhxwzdm8sj0rumuz8zj32slawx0hzc7rsq3hk5ns

This repo is powered by the gitpay daemon and LND. Get paid for closing issues
<br/>
Learn more about LND at https://dev.lightning.community/

### Steps to Test

1. Git clone, npm i 
2. run gitpayd in https mode
3. Verify pr is merged and payment sent

### Screenshots

[Only if applicable]
